docs/ProjectPlanning.md

```diff
@@ -17,43 +17,63 @@
   - unique database feature(s)
   - batching and queuing
   - [ACID](https://en.wikipedia.org/wiki/ACID) vs [BASE](https://en.wikipedia.org/wiki/Eventual_consistency)
-- School composition [MCSS Schools](https://mcssliberia.org/schools/)
+- School system composition [MCSS Schools](https://mcssliberia.org/schools/)
   - Are there school districts?
 - Do teachers:
   - Share classrooms?
   - Specialize(do kids change classrooms for different classes?) or generalize(teach same students all day)?
-  - Teach multiple groups if children ? (A day, B day)
+  - Do most teachers have mobile phones? 
+  - Teach multiple groups of children ? (A day, B day)
 - Is attendance for just classes or should it include exams?
+- Should attendance entry for particular classes be limited to a time-frame? (i.e. Teacher has 24 hours from time lesson occurs to enter attendance data. I figure this would prevent sham data proliferating and encourage same-day attendance taking)
+- Do teachers have a process to manually take attendnace currently? If not, how do we make it easy for them to remember to enter attendance data (e.g. push notifications)?
+
+## Required Functionality
+- Can query for Attendance statistics based on school, district, (etc.)?
+  --> Maintain DB with student attendance data
+
+## End Users
```

Comment:
These people would use the admin interface which comes with the back-end. Good to include.